specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-03-01-preview/connectedClusters.json

```diff
@@ -0,0 +1,802 @@
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "Identity for the connected cluster."
+    },
+    "ConnectedClusterProperties": {
+      "type": "object",
+      "required": [
+        "agentPublicKeyCertificate"
+      ],
+      "properties": {
+        "agentPublicKeyCertificate": {
+          "type": "string",
+          "description": "Base64 encoded public certificate used by the agent to do the initial handshake to the backend services in Azure."
+        },
+        "kubernetesVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Kubernetes version of the connected cluster resource"
+        },
+        "totalNodeCount": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "Number of nodes present in the connected cluster resource"
+        },
+        "totalCoreCount": {
+          "readOnly": true,
+          "format": "int32",
+          "type": "integer",
+          "description": "Number of CPU cores present in the connected cluster resource"
+        },
+        "agentVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Version of the agent running on the connected cluster resource"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ConnectedClusterProvisioningState",
+          "description": "Provisioning state of the connected cluster resource."
+        },
+        "distribution": {
+          "type": "string",
+          "description": "The Kubernetes distribution running on this connected cluster."
+        },
+        "infrastructure": {
+          "type": "string",
+          "description": "The infrastructure on which the Kubernetes cluster represented by this connected cluster is running on."
+        },
+        "offering": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Connected cluster offering"
+        },
+        "managedIdentityCertificateExpirationTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Expiration time of the managed identity certificate"
+        },
+        "lastConnectivityTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "Time representing the last instance when heart beat was received from the cluster"
+        },
+        "connectivityStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Represents the connectivity status of the connected cluster.",
+          "enum": [
+            "Connecting",
+            "Connected",
+            "Offline",
+            "Expired"
+          ],
+          "x-ms-enum": {
+            "name": "ConnectivityStatus",
+            "modelAsString": true
+          }
+        },
+        "systemData": {
```

Comment:
systemData exists in the ConnectedCluster definition (above this). It should be removed here